### PR TITLE
Categories And Medicines Migrations And Seeding

### PR DESCRIPTION
**Pull request by Thet-Htar-Soe**
## Link to Issue
task18: categories and medicines migrations and seeding

## What you did
categories and medicines migrations and seeding

## What not to do
There's none.

## Operation check
Check in browser: MYSQL workbence

## Evidence
image
medicines fake 30 datas 
image
categories fake 30 datas

## Other
There's none.

### CODE REVIEW

File: database/factories/MedicineFactory.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public function definition()
+    {
+        return [
+        'name'=>$this->faker->name(),
+        'classification'=>$this->faker->sentence(3),
+        'symptom'=>$this->faker->text(100),
+        'treatment'=>$this->faker->text(100),
+        'quantity'=>$this->faker->randomDigit(),
+        'price'=>$this->faker->numberBetween(100, 1000)
+        ];
```
Comment (YanKyawMinGIT) [inline database/factories/MedicineFactory.php:23]: use Tab key!

Comment (Thet-Htar-Soe) [reply database/factories/MedicineFactory.php:23]: hote kae fix lite pr p shint

File: database/factories/UserFactory.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function definition()
             'email_verified_at' => now(),
             'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
             'remember_token' => Str::random(10),
-        ];
+        ];
```
Comment (YanKyawMinGIT) [inline database/factories/UserFactory.php:19]: That is not necessary!

Comment (Thet-Htar-Soe) [reply database/factories/UserFactory.php:19]: remove lite pr p shint

**Review by YanKyawMinGIT — APPROVED** [review]

**Pull request merged**